File: Exercícios/Scripts_C#/ScriptsUsandoC#/ScriptsUsandoC#/Desafio_06.cs
public class Desafio_06
{
    public static void DobroTriploRaiz()
    {
        Console.Write("Digite um número: ");
        double numero = double.Parse(Console.ReadLine().Trim());

        double dobro = numero * 2;
        double triplo = numero * 3;
        double raiz = Math.Sqrt(numero);

        Console.WriteLine($"O dobro de {numero} é {dobro}");
        Console.WriteLine($"O triplo de {numero} é {triplo}");
        Console.WriteLine($"A raiz quadrada de {numero} é {raiz}");
    }
}

File: Exercícios/PythonMundo_1_2_3/Desafio_06.py
# Exercício Python 006: Crie um algoritmo que leia um número e mostre o seu dobro, triplo e raiz quadrada.

print('{} DESAFIO 006 {}'.format('='*6, '='*6))

numero = float(input('Digite um número: '))

dobro = numero * 2
triplo = numero * 3
raiz = numero ** (1/2)

print(f'\n[!] NÚMERO DIGITADO = {numero}\n[!] DOBRO = {dobro}\n[!] TRIPLO = {triplo}\n[!] RAIZ QUADRADA = {raiz:.2f}')
print('='*24)